### ISSUE TITLE
Incorrect example for Url.Parser.Query.map function.

### ISSUE THREAD

**Issue by akoppela** [elm/url#38]
https://package.elm-lang.org/packages/elm/url/latest/Url-Parser-Query#map

Example is:
```
page : Parser Int
page =
  map (Result.withDefault 1) (int "page")
```

According to signature for `int` function it should be
```
page : Parser Int
page =
  map (Maybe.withDefault 1) (int "page")
```

**Comment by evancz** [comment]
Fixed by #13

**Comment by akoppela** [comment]
Great work!
